internal/config/cgmgr/cgmgr.go

```diff
@@ -147,11 +147,7 @@ func createSandboxCgroup(sbParent, containerID string, mgr CgroupManager) error
 	if err != nil {
 		return err
 	}
-	if mgr.IsSystemd() {
-		_, err = cgroups.NewSystemd(path, &cgcfgs.Resources{})
-	} else {
-		_, err = cgroups.New(path, &cgcfgs.Resources{})
-	}
+	_, err = cgroups.New(path, &cgcfgs.Resources{})
```

Comment:
what do you think about moving this whole function to cgroupfs.go? it seems odd to keep the function generic if it's only used specifically

Reply:
@n4j were you planning on this?

Reply:
@haircommander yes I am, I'll submit a PR in a day or two

Reply:
are you opposed to updating this PR?

Reply:
@haircommander Oh not all, was caught up in some stuff so couldn't look at it.